go/src/heif-test.go

```diff
@@ -20,7 +20,6 @@
 
 package main
 
-
 /*
 #cgo CPPFLAGS: -I/home/domain/farindk/sys/include/libheif
```

Comment:
While you're at it, this should be the same as in https://github.com/strukturag/libheif/blob/8a3020dc8d26b18f2f5bc202af5024086281b7e4/go/src/heif/heif.go#L24
e.g. without the hardcoded include path and updated `#include <libheif/heif.h>`

Reply:
#61